internal/page/certificateprovider/enter_reference_number.go

```diff
@@ -52,17 +52,13 @@ func EnterReferenceNumber(tmpl template.Template, shareCodeStore ShareCodeStore,
 					LpaID:     shareCode.LpaKey.ID(),
 				})
 
-				if _, err := certificateProviderStore.Create(ctx, shareCode.LpaOwnerKey, shareCode.ActorUID, session.Email); err != nil {
+				if _, err := certificateProviderStore.Create(ctx, shareCode, session.Email); err != nil {
```

Comment:
the error handling below will need updating, which looks like it might get annoying:

```go
var ex *types.TransactionCanceledException
if !errors.As(err, &ex) {
  return err
}
for _, reason := range ex.CancellationReasons {
  if reason.Code != aws.String("ConditionalCheckFailed") {
    return err
  }
}
```

So may end up being better to wrap in `WriteTransactions` and expose a simpler error for here